src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx

```diff
@@ -49,6 +49,38 @@ To [view your app's performance in the New Relic UI](/docs/apm/applications-menu
 
 You can also [extend your agent instrumentation](#extend-instrumentation) and explore your data with other New Relic products, including New Relic One's [data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) and [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-new-relic-one-dashboards).
 
+## Evaluate our Java agent with guided install [#guided-install]
+
+Our guided install creates a customized CLI command for your environment that downloads and installs the Java agent. 
+
+<Callout variant="caution">
+The guided install version of the Java agent is for evaluation purposes only. It will not persist when you restart your Java application.
```

Comment:
Can we had some friendlier language than "for evaluation purposes only"? That seems like it's discouraging me from using it. Maybe something like, "The guided install lets you test out data with our Java agent, without retaining any permanent data. The data will not remain when you restart your Java application."?

Reply:
I agree, your language is a lot friendlier :)